tests/unit/subscribeToPropertiesTest.tsx

```diff
@@ -138,7 +139,7 @@ describe('Only the specific property changes when using withOnyx() and ', () =>
                     Onyx.mergeCollection(ONYX_KEYS.COLLECTION.TEST_KEY, {
                         [`${ONYX_KEYS.COLLECTION.TEST_KEY}1`]: {a: 'one', b: 'two'},
                         [`${ONYX_KEYS.COLLECTION.TEST_KEY}2`]: {c: 'three', d: 'four'},
-                    }),
+                    } as Collection<string, unknown, unknown>),
```

Comment:
`Collection<string, unknown, unknown>` is used a lot, maybe it makes sense to create a named type for it